qregister.hpp

```diff
@@ -291,81 +291,81 @@ class CoherentUnit {
      * Add integer of "length" bits in "inStart" to integer of "length" bits in "inOutStart," and store result in
      * "inOutStart."
      */
-    virtual void ADD(const bitLenInt inOutStart, const bitLenInt inStart, const bitLenInt length);
+    //virtual void ADD(const bitLenInt inOutStart, const bitLenInt inStart, const bitLenInt length);
```

Comment:
If these are truly dead, then delete the code outright.  It can always be pulled from the git history if it’s needed again!

Reply:
Sorry, no, they're not dead. They're referenced in the README as the basis for near-future correct implementations. The overwriting in the ADD/SUB variants is just quantum mechanically wrong, but they're not far from correct implementations. Hence, I was thinking they should be temporarily removed.